benchexec/tools/moxi-mc-flow.py

```diff
@@ -0,0 +1,52 @@
+# This file is part of BenchExec, a framework for reliable benchmarking:
+# https://github.com/sosy-lab/benchexec
+#
+# SPDX-FileCopyrightText: 2007-2020 Dirk Beyer <https://www.sosy-lab.org>
+#
+# SPDX-License-Identifier: Apache-2.0
+
+import benchexec.result as result
+import benchexec.tools.template
+from math import ceil
+
+
+class Tool(benchexec.tools.template.BaseTool2):
+    """
+    Tool info for MoXI-MC-Flow
+    """
+
+    REQUIRED_PATHS = [
+        "deps/",
+        "src/",
+        "json-schema/",
+        "sortcheck.py",
+        "translate.py",
+    ]
+
+    def executable(self, tool_locator):
+        return tool_locator.find_executable("modelcheck.py")
+
+    def name(self):
+        return "MoXI-MC-Flow"
+
+    def project_url(self):
+        return "https://github.com/ModelChecker/moxi-mc-flow"
+
+    def program_files(self, executable):
+        return self._program_files_from_executable(executable, self.REQUIRED_PATHS)
```

Comment:
Is it deliberate that this overwrites our default implementation and only changes it to skip adding the `executable`?

Reply:
Fixed in e1fee24